### DISCUSSION TITLE
Alternatives to AbsoluteLayout since it's been removed

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/maui#2095] [Q&A] [answered]
I'm a bit confused about the removal of AbsoluteLayout, what are the alternatives to using AbsoluteLayout for absolute positioning an element? Or when positioning an element absolutely on top of other elements for example when displaying an overlay? Is there some other element that takes its place?

Also i haven't been able to run the WinUI project i keep getting the message:

"Unable to start debugging. The startup project cannot be launched. Ensure that the correct project is set as the startup project. The startup project can be changed by selecting the 'Set as startup project' command fromt he right click menu in solution explorer. Additionally make sure its debug settings are correctly configured in project properties."

I'm using the default configuration from the dropdown menu that has the name project_name.WinUI and some suggestions online have been to click deploy, and that seems to build fine but the application doesn't run. It's a fresh install of windows 10 and developer mode is enabled but i guess i could be making some other basic mistake. I have followed the installation instructions and have project reunion and msix installed.

**Comment by hartez** [comment, marked as answer]
AbsoluteLayout should still be available in the Compatibility namespace. There may be a bug where the renderer is not found at runtime; once that's addressed, AbsoluteLayout will be available. 

And a new AbsoluteLayout using the new layout system is in the works, though I don't know when that will be available. Regardless, it's not gone forever. Just temporarily misplaced.

**Reply by ghost** [reply]
Alright good to know, thanks.

**Reply by hartez** [reply]
Here's the relevant bug: https://github.com/dotnet/maui/issues/2062
